### PR TITLE
Edit Services document

### PR DESCRIPTION
**Pull request by willingc**
@minrk @jhamrick I've done some pretty significant editing to the Services document to be more user focused. I've tried not to change the meaning of Min's original spec but please do double-check that I haven't misinterpreted anything.

### CODE REVIEW

File: docs/source/services.md [hunk 1]
```diff
@@ -1,70 +1,115 @@
...
+### Properties of a Service
+
+If a `command` is specified for launching the Service, the Service will be
+started and managed by the Hub.
+
+If a `url` is specified for where the Service runs its own web server,
+JupyterHub will add the Service to the Hub's proxy at 
+`/service/:service-name`.
+
+## Managed Service
```
Comment (willingc) [inline docs/source/services.md]: Should we change this to 'Hub-managed Service'?

As well as change 'External Service' to 'Externally-managed Service' or 'Self-managed Service'? Or simply leave as 'External Service'?

Comment (minrk) [inline docs/source/services.md]: Yes, I think Hub-Managed and Externally-Managed is clearer.

**Comment by minrk** [comment]
The changes are great, thanks @willingc!

**Comment by willingc** [comment]
Updated naming to 'Hub-Managed' and 'Externally-Managed'.

Corrected meaning of 'admin' in Line 55.

**Comment by minrk** [comment]
Awesome, thanks @willingc! 🍰

**Pull request merged**